README.md

```diff
@@ -283,6 +287,43 @@ Fill in the missing pieces in [this](https://github.com/Azure/kubernetes-keyvaul
 
   > **NOTE**: When using the `Pod Identity` option mode, there may be some delay in obtaining the objects from Key Vault. During pod creation time, AAD Pod Identity needs to create the `AzureAssignedIdentity` for the pod based on the `AzureIdentity` and `AzureIdentityBinding` and retrieve the token for Key Vault. It is possible for the pod volume mount to fail during this time. If it does, the kubelet will keep retrying until after the token retrieval is complete and the mount succeeds.
 
+#### OPTION 3: VMSS User Assigned Managed Identity
```

Comment:
we probably need another section for `VMSS System Assigned Managed Identity` but I can add that later as a followup PR.

Reply:
also, for `usepodidentity` cases where the VMSS has system assigned identities, we need to perform the following:

```
az role assignment create --role "Managed Identity Operator" --assignee <VMSS SYSTEM ASSIGNED IDENTITY OBJECTID> --scope <USER ASSIGNED IDENTITY RESOURCE ID>
```